snorkel/labeling/model/label_model.py

```diff
@@ -334,7 +336,7 @@ def get_conditional_probs(self) -> np.ndarray:
         np.ndarray
             An [m, k + 1, k] np.ndarray conditional probabilities table.
         """
-        return self._get_conditional_probs(self.mu.detach().numpy())
+        return self._get_conditional_probs(self.mu.clone().cpu().detach().numpy())
```

Comment:
Why is clone() necessary here but not the other places where you're adding `.cpu().detach()`?

Reply:
good catch, probably added during debugging. removed